picard/ui/tagsfromfilenames.py

```diff
@@ -165,3 +168,6 @@ def accept(self):
         config = get_config()
         config.persist["tags_from_filenames_format"] = self.ui.format.currentText()
         super().accept()
+
+    def help(self):
+        webbrowser2.goto('doc_tagsfromfilename')
```

Comment:
The key is `doc_tags_from_filenames` above

Reply:
Oh sorry, forgot to include a local change. Fixed now